core/coreapi/pin_test.go

```diff
@@ -145,7 +147,7 @@ func TestPinRecursive(t *testing.T) {
 		t.Error("unexpected path")
 	}
 
-	list, err = api.Pin().Ls(ctx, api.Pin().WithType("indirect"))
+	list, err = api.Pin().Ls(ctx, opt.Pin.Type("indirect"))
```

Comment:
Nicer UX would be if we had already defined functions for different pin types. Like `opt.Pin.TypeIndirect` or `opt.Pin.Type.Indirect`

Reply:
Done